lib/license_finder/package_managers/gradle.rb

```diff
@@ -42,8 +42,26 @@ def package_management_command
       File.exist?(File.join(project_path, wrapper)) ? wrapper : gradle
     end
 
+    def active?
+      result = super
+
+      # If it is a valid gradle module, is it a root module?
+      result = root_module? if result
```

Comment:
My only concern around this check is that if the user is inside a gradle subproject, running license_finder will say it does detect gradle which can be confusing for users. If we are going with this change, i would recommend a warning message describing why license_finder did not recognize gradle package manager inside a gradle subproject directory.

Reply:
Also, i would recommend confirming with Kim as this is a little bit out of scope of the story.